src/app/views/contact_view.js

```diff
@@ -1,6 +1,39 @@
+import $ from 'jquery';
 import Backbone from 'backbone';
+import _ from 'underscore';
+import Contact from 'app/models/contact';
+
 
 const ContactView = Backbone.View.extend({
+  initialize: function(options) {
+    this.contact = options.contact;
+    this.template = options.template;
+    // this.listenTo(this.model, "change", this.render);
+  },
+  render: function() {
+  // reconnects the DOM event Handlers
+  this.delegateEvents();
+  //could use .toJSON() instead of .attributes
+  var html = this.template(this.model.attributes);
+  this.$el.html(html);
+  // Enable chained calls
+  // This is important enough that we'll leave it in, but
+  // we wont talk about it until later.
+  return this;
+  },
+  events: {
+    'click .contact-card': 'showModal'
+  },
+  showModal: function() {
+    var detailsTemplate = _.template($('#tmpl-contact-details').html());
```

Comment:
This works and is fine here, but it's best practice to only render things within the view, not things outside the view.  

I'd suggest a better method is `triggering` an event and responding to it in the Rolodex View.